.nvmrc

```diff
@@ -0,0 +1 @@
+v18.16.0
```

Comment:
@MattiaPrimavera 

⚠️  current deployment CI/CD pipeline (and its associated Docker image) is based onto Node `v18.18'

https://github.com/noosenergy/noos-docker-images/blob/master/docker/circleci/Dockerfile#L22

Reply:
> ⚠️ Current CI/CD pipeline (and its associated Docker image) is based onto Node v18.18.

thanks for the precision, I'll update the file accordingly

> However such a file could be added onto the project .gitignore as the Node version constraint is already captured within the library package.kson config as well as in the devcontainer Docker image.

I think `.nvmrc` could serve a different purpose and is normally meant to be versioned : it can allow easily installing the correct node version on local environment (`nvm i` & `nvm use` commands) for developers using `nvm`, and it provides more precisely than the `package.json` the specific node version to use :

```js
  // package.json
  "engines": {
    "node": ">=18"
  },
```

> as well as in the devcontainer Docker image

Isn't that information specified in the base docker image although ? I see no "node" precise version if I search through this projects files

Reply:
Yes, I agree that for the developer is quite convenient to have this file.

Reply:
I've updated node version in `.nvmrc` file